поправил fullName стал String

homebrewdev opened the pull request:
поправил в структуре struct Person
fullName стал String

homebrewdev committed:
поправил fullName стал String

homebrewdev committed:
решение задач №3 и № 4

homebrewdev committed:
фигуры в задаче 4 добавил в словарь и циклом вывожу shapeDescription

OldSmith commented on week2/myStructClass.swift:
```diff
@@ -46,5 +46,84 @@ struct Person {
+    
+    override func shapeDescription() -> String {
+        return "shape is \(name)"
+    }
```
не зря в super классе метод shapeDescription уже возвращет "shape is" - подумай как это использовать тут

homebrewdev replied:
поправил все
в задаче 4 про фигуры все поправил - shapeDescription используется в Shape
использую super.init()
площадь - вычисляемое свойство каждой фигуры
фигуры добавлены в массив и выводится статистика по их дескрипшенам и площади

OldSmith commented on week2/main.swift:
```diff
@@ -150,3 +150,29 @@ print(persona.fullName) //Bro John
+
+print(sqr.shapeDescription()) // shape is Square
+print(circle.shapeDescription()) // shape is Circle
+
```
Создать несколько разных экземпляров каждого класса и поместить их в array

homebrewdev replied:
фигуры добавлены в массив и выводится статистика по их дескрипшенам и площади

OldSmith commented on week2/myStructClass.swift:
```diff
@@ -46,5 +46,84 @@ struct Person {
+        }
+    }
+    subscript (insert index: Int, character char: Character) -> Bool {
+        get{return true}
```
переделай, чтобы вызов subscript выглядел exmpl[0] и exmpl[1] = "8"

homebrewdev committed:
в задаче 4 про фигуры все поправил - shapeDescription используется в Shape
использую super.init()
площадь - вычисляемое свойство каждой фигуры
фигуры в массиве и выводится статистика по их дескрипшенам и площади

в задаче 3 поправил сабскрипт
